pkg/tlscheck/tlscheck.go

```diff
@@ -46,8 +46,8 @@ func CheckTLS(ctx context.Context, origAddr string) (bool, error) {
 		case x509.CertificateInvalidError,
 			x509.HostnameError,
 			x509.UnknownAuthorityError,
-			tls.RecordHeaderError,
-			*tls.CertificateVerificationError:
+			tls.RecordHeaderError:
+			//*tls.CertificateVerificationError:
```

Comment:
unintended?

Reply:
Yes this is some weirdness in my goland setup. It doesn't let me run tests locally - givens me compile errors until I comment this out. Please revert.